### DISCUSSION TITLE
Error: The result of this StaticQuery could not be fetched.

### DISCUSSION THREAD

**Discussion by kuroroblog** [gatsbyjs/gatsby#33920] [Help]
# Overview

I run a personal site using gatsby.
my site : https://kuroro.blog/

The following errors may occur while operating the service.See image below.

<img width="1680" alt="screenshot 2021-11-10 15 32 00" src="https://user-images.githubusercontent.com/23373288/141065722-057f5d40-522b-4d54-a993-c3c8f5a9c411.png">

## Steps to Reproduce

1. Use google chrome.
2. Visit my site(https://kuroro.blog/).
3. Click some links.
4. Occasionally the above error occurs.(※ Works fine in secret mode.)

## Expected Result

<img width="1680" alt="screenshot 2021-11-10 16 10 44" src="https://user-images.githubusercontent.com/23373288/141067257-b8c6ed16-1229-4089-9712-ece313eb7d17.png">

## Actual Result

<img width="1680" alt="screenshot 2021-11-10 16 10 13" src="https://user-images.githubusercontent.com/23373288/141067137-6874d611-57b5-40fc-9ca0-36cade2b3c87.png">

## Environment

OS : MacOS
my packages 

"dependencies": {
    "firebase": "^8.6.7",
    "gatsby": "^3.6.2",
    "gatsby-plugin-advanced-sitemap": "^2.0.0",
    "gatsby-plugin-clarity": "^1.0.0",
    "gatsby-plugin-dark-mode": "^1.1.2",
    "gatsby-plugin-google-adsense": "^1.1.3",
    "gatsby-plugin-google-gtag": "^3.7.1",
    "gatsby-plugin-manifest": "^3.6.0",
    "gatsby-plugin-offline": "^4.6.0",
    "gatsby-plugin-react-helmet": "^4.6.0",
    "gatsby-plugin-twitter": "^3.12.0",
    "gatsby-source-filesystem": "^3.7.1",
    "react": "^17.0.1",
    "react-calendar-heatmap": "^1.8.1",
    "react-copy-to-clipboard": "^5.0.4",
    "react-dom": "^17.0.1",
    "react-helmet": "^6.1.0",
    "react-lazyload": "^3.2.0",
    "react-markdown": "5.0.3",
    "react-scroll": "^1.8.2",
    "react-slick": "^0.28.1",
    "react-syntax-highlighter": "^15.4.3",
    "react-tooltip": "^4.2.21",
    "remark-gfm": "^1.0.0",
    "slick-carousel": "^1.8.1",
    "typescript": "^4.3.2"
  },

useStaticQueryCode

``` js
const data = useStaticQuery(
        graphql`
          query {
            allArticle(sort: { order: DESC, fields: updatedAt___seconds }) {
              edges {
                node {
                  category
                  description
                  id
                  thumbnail
                  title
                  article
                  createdAt {
                    seconds
                  }
                  updatedAt {
                    seconds
                  }
                }
              }
            }
          }
        `
      )
```

**Comment by kim-lindgren** [comment]
Hi Kuroro,

Did you find a solution? I have the same problem. In my case, the error prevents my site from being loaded completely on my local server.
